src/main/java/org/jivesoftware/smackx/pubsub/PubSubIntegrationTest.java

```diff
@@ -915,4 +923,44 @@ public void transientNotificationOnlyNodeWithItemTest() throws NoResponseExcepti
             pubSubManagerOne.deleteNode(nodename);
         }
     }
+
+    @SmackIntegrationTest
+    public void isEqualToOneTest() {
+        assertEquals(1,1);
+    }
+
+    @SmackIntegrationTest
+    public void sendMessageEventTest() throws NoResponseException, XMPPErrorException, NotConnectedException, InterruptedException, XmppStringprepException, TestNotPossibleException {
+        final String nodename = "sinttest-publish-item-nodename-" + testRunId;
+        final String needle = "test content " + Math.random();
+        LeafNode publisherNode = pubSubManagerOne.createNode(nodename);
+        try {
+            // Publish a new item.
+            publisherNode.publish( new PayloadItem<>( GeoLocation.builder().setDescription( needle ).build() ) );
+            Node subscriberNode = pubSubManagerTwo.getNode(nodename);    
+            ChatManager chatManager = ChatManager.getInstanceFor(connection);
+            final EntityBareJid subscriber = conTwo.getUser().asEntityBareJid();
+            subscriberNode.subscribe( subscriber );
+            Message m = new Message();
```

Comment:
Up until this line, I think I know what happens:
- user 1 creates a pubsub node
- user 1 publishes an item (with a geolocation item) to the newly created node.
- user 2 subscribes to the node

Now, things get unclear for me. The code obtains a ChatManager, and constructs a Chat Message with a Message Event, and sends the chat with the subscriber. I don't understand why you do this. It might be that I do not understand the purpose of this test (which is why it is important to document that in the javadoc!) or you might not fully understand how you should use Smack (which is perfectly understandable - I'm here to help with that!).

Chat messages are quite different from Pubsub notifications. What you send is a stanza like the snippet below. As you can see, that does not include any pubsub data. Is this what you intended?

```<message to="smack-inttest-two-s13ns@laptop-guus" id="EAA3G-166" from="smack-inttest-one-s13ns@laptop-guus/one-s13ns">
  <x xmlns="jabber:x:event">
    <offline/>
    <id>
      EAA3G-167
    </id>
  </x>
</message>```